src/js/ia/IAPage.js

```diff
@@ -360,6 +361,10 @@
                         }
                     });
 
+                    $("body").on('click', ".userpage-link a", function(evt) {
```

Comment:
@bsstoner Most (if not all) of the IA Pages elements are dynamically created: after page load, we call [this function](https://github.com/duckduckgo/community-platform/blob/maria/replace-urls/src/js/ia/IAPage.js#L1810) which renders the Handlebars templates associated with each piece of the page in the desired order. This function is also used to refresh the page when editing a field.

This means binding the events to the elements directly [won't work](http://stackoverflow.com/questions/13650261/jquery-cannot-bind-events-to-dynamic-elements), since they previously didn't exist, while binding on body and then seeing if the specific element clicked is the right one works.

Something I learned in my early web dev days :smile: